### ISSUE TITLE
Pause at layer not working

### ISSUE THREAD

**Issue by donhohler** [bambulab/BambuStudio#5253]
### Bambu Studio Version

1.10.0.89

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 10

### Printer

X1 Carbon

### How to reproduce

I add a pause at any layer. Then slice the file. Examine the g-code. There is no pause. This just started happening with the latest update.

### Actual results

When I print the file there is no pause.

### Expected results

There should have been a pause.

### Project file & Debug log uploads

[Intake Bracket.zip](https://github.com/user-attachments/files/17742291/Intake.Bracket.zip)

### Checklist of files to include

- [ ] Log file
- [X] Project file

**Comment by donhohler** [comment]
I also tried adding custom g-code. M400 U1. Nothing. It did not get added to the g-code.

**Comment by donhohler** [comment]
I reverted back to 1.9.7.52 and everything works.

**Comment by MartinChan3** [comment]
![image](https://github.com/user-attachments/assets/ac9300a5-bcd7-4865-815f-c5af811cc533)
It is the same, it seems that all things are right with my Studio and X1C...

**Comment by donhohler** [comment]
There is definitely a problem. I reinstalled 1.10 and got the same problem again.

**Comment by MartinChan3** [comment]
I believe Bambu will fix this issue in next version.

**Comment by donhohler** [comment]
Yes. I could not connect to my printer with it. No matter what I tried. I'm back to 1.9 again.

**Comment by DavidLorentzen** [comment]
I have 1.10.1 and experienced the same problem, just an hour ago. The first print paused correctly, and I imported another object of the same specs, and pause was skipped. The pause was supposed to be at the same layer (19) as the first. Both prints were imported as 3mf files.

**Comment by XunZhangBambu** [comment]
@DavidLorentzen Could you please provide the 3mf?

**Comment by donhohler** [comment]
The pause problem was supposedly fixed in the latest version. However, when I tried it, it would not connect to the printer. So, I can not confirm whether it's fixed or not. I am currently still using ver 1.9. It would be nice if Bambu would do a little in house testing before throwing these updates out in the wild for all of us to test for them.

**Comment by DavidLorentzen** [comment]
[Aaron-3mf.3mf.txt](https://github.com/user-attachments/files/17808200/Aaron-3mf.3mf.txt)

I added a .txt to the end of the file name to upload.

Dave

**Comment by DavidLorentzen** [comment]
Encountered the problem again yesterday, different file. I set a pause in order to insert a nut. It blew right past the pause and ran to the end of the print. STL is attached.

[Pivot+Handle+-+SAE+-+Quarter20.stl.txt](https://github.com/user-attachments/files/17894304/Pivot%2BHandle%2B-%2BSAE%2B-%2BQuarter20.stl.txt)

**Comment by XunZhangBambu** [comment]
@DavidLorentzen  
Hello, I tried the file you provided and added the pause G-code, but I couldn't reproduce your issue. You should be able to directly provide the 3MF file with parameters without needing to change the file extension.

![image](https://github.com/user-attachments/assets/d3714574-f663-4ffd-bea7-b741a4d2a03a)
![image](https://github.com/user-attachments/assets/b7199462-59e0-4808-8bde-6d0b59a769f5)

**Comment by DavidLorentzen** [comment]
Sorry, I've been working with STL files, and didn't save all my settings as 3MF. I've tried unsuccessfully to recreate the problem after two attempts.
